target/linux/ramips/dts/mt7621_pangubox_pgb-m1.dts

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible =  "PanguBox,PGB-M1","mediatek,mt7621-soc";
+	model = "PanguBox PGB-M1";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led_power_green: sys {
+			label = "blue:sys";
```

Comment:
Honestly? led_power_green for blue:sys?

I'll wait until this whole thing is ready for presentation.

Reply:
I'm sorry, I will check again before submitting next time.I am trying to commit pr to OpenWRT and may make some silly mistakes.Thank you for pointing out my problems in your busy schedule.